scripts/commands/sanction.lua

```diff
@@ -0,0 +1,20 @@
+---------------------------------------------------------------------------------------------------
+-- func: Sanction
+-- desc: 
+---------------------------------------------------------------------------------------------------
+require("scripts/globals/besieged")
+
+
+cmdprops =
+{
+    permission = 0,
```

Comment:
If server operators want to let their players give themselves sanction/signet etc. at will, that's up to them, but it shouldn't be part of the commands available to players by default.